RepositoryCompiler/CodeModel/CaDETModel/Metrics/CaDETClassMetrics.cs

```diff
@@ -9,5 +9,47 @@ public class CaDETClassMetrics
         public int WMC { get; set; }
         public int ATFD { get; set; }
         public double? TCC { get; set; }
+
+        /// <summary>
+        /// NOR: Number of return statements
+        /// Implementation based on https://github.com/mauricioaniche/ck
```

Comment:
These comments should be tied to the function that does the calculation - if we change the implementation, we won't be changing this class, but the metric calculator one.